src/algorithms/pid/MergeParticleID.cc

```diff
@@ -0,0 +1,199 @@
+// Copyright 2023, Christopher Dilks
+// Subject to the terms in the LICENSE file found in the top-level directory.
+
+#include "MergeParticleID.h"
+
+// AlgorithmInit
+//---------------------------------------------------------------------------
+void eicrecon::MergeParticleID::AlgorithmInit(std::shared_ptr<spdlog::logger>& logger)
+{
+  m_log = logger;
+  m_cfg.Print(m_log, spdlog::level::debug);
+}
+
+
+// AlgorithmChangeRun
+//---------------------------------------------------------------------------
+void eicrecon::MergeParticleID::AlgorithmChangeRun() {
+}
+
+
+// AlgorithmProcess
+//---------------------------------------------------------------------------
+std::unique_ptr<edm4eic::CherenkovParticleIDCollection> eicrecon::MergeParticleID::AlgorithmProcess(
+    std::vector<const edm4eic::CherenkovParticleIDCollection*> in_pid_collections_list
+    )
+{
+  // logging
+  m_log->trace("{:=^70}"," call MergeParticleID::AlgorithmProcess ");
+
+  // start output collection
+  auto out_pids = std::make_unique<edm4eic::CherenkovParticleIDCollection>();
+
+  /* match input `CherenkovParticleIDCollection` elements from each list of
+   * collections in `in_pid_collections_list`
+   * - the matching is done by the ID of the charged particle object
+   * - each unique charged particle object ID should correspond to a unique
+   *   charged particle object, so any "merging" of the charged particle
```

Comment:
```suggestion
   *   charged particle object, so any matching of the charged particle
```
If I understand correctly

Reply:
"merging" is correct here: it means merging of the radiators' `TrackPoint`s.